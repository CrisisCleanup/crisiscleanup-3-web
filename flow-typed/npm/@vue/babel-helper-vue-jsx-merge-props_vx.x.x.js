// flow-typed signature: d7a23c53f4091ef2bf1df8afa5eb43c4
// flow-typed version: <<STUB>>/@vue/babel-helper-vue-jsx-merge-props_v^1.0.0/flow_v0.131.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/babel-helper-vue-jsx-merge-props'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/babel-helper-vue-jsx-merge-props' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/babel-helper-vue-jsx-merge-props/dist/helper' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/babel-helper-vue-jsx-merge-props/dist/helper.js' {
  declare module.exports: $Exports<'@vue/babel-helper-vue-jsx-merge-props/dist/helper'>;
}
