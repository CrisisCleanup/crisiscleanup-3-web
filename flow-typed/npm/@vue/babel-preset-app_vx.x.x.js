// flow-typed signature: 6ba91a92845d9a556a5bcdcc4f033fed
// flow-typed version: <<STUB>>/@vue/babel-preset-app_v^4.3.1/flow_v0.129.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/babel-preset-app'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/babel-preset-app' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/babel-preset-app/polyfillsPlugin' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/babel-preset-app/index' {
  declare module.exports: $Exports<'@vue/babel-preset-app'>;
}
declare module '@vue/babel-preset-app/index.js' {
  declare module.exports: $Exports<'@vue/babel-preset-app'>;
}
declare module '@vue/babel-preset-app/polyfillsPlugin.js' {
  declare module.exports: $Exports<'@vue/babel-preset-app/polyfillsPlugin'>;
}
