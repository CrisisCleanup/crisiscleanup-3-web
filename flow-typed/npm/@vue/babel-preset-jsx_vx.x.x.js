// flow-typed signature: eeecaeff29e4a3a6c63c4fe7e16a0eb3
// flow-typed version: <<STUB>>/@vue/babel-preset-jsx_v^1.1.2/flow_v0.131.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/babel-preset-jsx'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/babel-preset-jsx' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/babel-preset-jsx/dist/plugin.cjs' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/babel-preset-jsx/dist/plugin.cjs.js' {
  declare module.exports: $Exports<'@vue/babel-preset-jsx/dist/plugin.cjs'>;
}
