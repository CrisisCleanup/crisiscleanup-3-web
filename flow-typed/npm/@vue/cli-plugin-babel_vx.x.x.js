// flow-typed signature: 4cf1a98e80b05c0a167e013d64bc9645
// flow-typed version: <<STUB>>/@vue/cli-plugin-babel_v^4.4.6/flow_v0.131.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/cli-plugin-babel'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/cli-plugin-babel' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/customConfig.input' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/customConfig.output' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/default.input' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/default.output' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/doubleQuote.input' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/doubleQuote.output' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/require.input' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/require.output' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/templateLiteral.input' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/templateLiteral.output' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/codemods/usePluginPreset' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/generator' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/migrator' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-babel/preset' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/customConfig.input.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/customConfig.input'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/customConfig.output.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/customConfig.output'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/default.input.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/default.input'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/default.output.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/default.output'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/doubleQuote.input.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/doubleQuote.input'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/doubleQuote.output.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/doubleQuote.output'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/require.input.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/require.input'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/require.output.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/require.output'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/templateLiteral.input.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/templateLiteral.input'>;
}
declare module '@vue/cli-plugin-babel/codemods/__testfixtures__/templateLiteral.output.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/__testfixtures__/templateLiteral.output'>;
}
declare module '@vue/cli-plugin-babel/codemods/usePluginPreset.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/codemods/usePluginPreset'>;
}
declare module '@vue/cli-plugin-babel/generator.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/generator'>;
}
declare module '@vue/cli-plugin-babel/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel'>;
}
declare module '@vue/cli-plugin-babel/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel'>;
}
declare module '@vue/cli-plugin-babel/migrator/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/migrator'>;
}
declare module '@vue/cli-plugin-babel/migrator/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/migrator'>;
}
declare module '@vue/cli-plugin-babel/preset.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-babel/preset'>;
}
