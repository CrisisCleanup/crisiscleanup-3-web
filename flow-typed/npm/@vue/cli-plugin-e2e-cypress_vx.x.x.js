// flow-typed signature: 122252e4d027555936ded476aff636b4
// flow-typed version: <<STUB>>/@vue/cli-plugin-e2e-cypress_v^4.4.6/flow_v0.129.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/cli-plugin-e2e-cypress'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/cli-plugin-e2e-cypress' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/cli-plugin-e2e-cypress/generator' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/_eslintrc' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/plugins' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/specs/test' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/support/commands' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/support' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-e2e-cypress/ui' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/cli-plugin-e2e-cypress/generator/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator'>;
}
declare module '@vue/cli-plugin-e2e-cypress/generator/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator'>;
}
declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/_eslintrc.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/_eslintrc'>;
}
declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/plugins/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/plugins'>;
}
declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/plugins/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/plugins'>;
}
declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/specs/test.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/specs/test'>;
}
declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/support/commands.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/support/commands'>;
}
declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/support/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/support'>;
}
declare module '@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/support/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/generator/template/tests/e2e/support'>;
}
declare module '@vue/cli-plugin-e2e-cypress/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress'>;
}
declare module '@vue/cli-plugin-e2e-cypress/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress'>;
}
declare module '@vue/cli-plugin-e2e-cypress/ui.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-e2e-cypress/ui'>;
}
