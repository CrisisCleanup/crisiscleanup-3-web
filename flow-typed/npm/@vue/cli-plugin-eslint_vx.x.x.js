// flow-typed signature: 190fd4925231c8aa9cbfe00ac40be4f1
// flow-typed version: <<STUB>>/@vue/cli-plugin-eslint_v^4.4.6/flow_v0.129.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/cli-plugin-eslint'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/cli-plugin-eslint' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/cli-plugin-eslint/eslintDeps' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-eslint/eslintOptions' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-eslint/generator' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-eslint/lint' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-eslint/migrator' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-eslint/prompts' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-eslint/ui/configDescriptor' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-eslint/ui' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-eslint/ui/taskDescriptor' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/cli-plugin-eslint/eslintDeps.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/eslintDeps'>;
}
declare module '@vue/cli-plugin-eslint/eslintOptions.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/eslintOptions'>;
}
declare module '@vue/cli-plugin-eslint/generator/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/generator'>;
}
declare module '@vue/cli-plugin-eslint/generator/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/generator'>;
}
declare module '@vue/cli-plugin-eslint/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint'>;
}
declare module '@vue/cli-plugin-eslint/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint'>;
}
declare module '@vue/cli-plugin-eslint/lint.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/lint'>;
}
declare module '@vue/cli-plugin-eslint/migrator/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/migrator'>;
}
declare module '@vue/cli-plugin-eslint/migrator/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/migrator'>;
}
declare module '@vue/cli-plugin-eslint/prompts.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/prompts'>;
}
declare module '@vue/cli-plugin-eslint/ui/configDescriptor.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/ui/configDescriptor'>;
}
declare module '@vue/cli-plugin-eslint/ui/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/ui'>;
}
declare module '@vue/cli-plugin-eslint/ui/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/ui'>;
}
declare module '@vue/cli-plugin-eslint/ui/taskDescriptor.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-eslint/ui/taskDescriptor'>;
}
