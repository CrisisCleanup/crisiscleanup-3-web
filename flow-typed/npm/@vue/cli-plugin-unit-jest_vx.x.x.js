// flow-typed signature: 345a830bc132c538d84f7a5ce68b6d33
// flow-typed version: <<STUB>>/@vue/cli-plugin-unit-jest_v^4.4.6/flow_v0.131.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@vue/cli-plugin-unit-jest'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@vue/cli-plugin-unit-jest' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */
declare module '@vue/cli-plugin-unit-jest/generator' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-unit-jest/generator/template/tests/unit/example.spec' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-unit-jest/jest-preset' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-unit-jest/presets/default/jest-preset' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-unit-jest/presets/no-babel/esmoduleTransformer' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-unit-jest/presets/no-babel/jest-preset' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-unit-jest/presets/typescript-and-babel/jest-preset' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-unit-jest/presets/typescript/jest-preset' {
  declare module.exports: any;
}

declare module '@vue/cli-plugin-unit-jest/ui' {
  declare module.exports: any;
}

// Filename aliases
declare module '@vue/cli-plugin-unit-jest/generator/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/generator'>;
}
declare module '@vue/cli-plugin-unit-jest/generator/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/generator'>;
}
declare module '@vue/cli-plugin-unit-jest/generator/template/tests/unit/example.spec.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/generator/template/tests/unit/example.spec'>;
}
declare module '@vue/cli-plugin-unit-jest/index' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest'>;
}
declare module '@vue/cli-plugin-unit-jest/index.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest'>;
}
declare module '@vue/cli-plugin-unit-jest/jest-preset.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/jest-preset'>;
}
declare module '@vue/cli-plugin-unit-jest/presets/default/jest-preset.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/presets/default/jest-preset'>;
}
declare module '@vue/cli-plugin-unit-jest/presets/no-babel/esmoduleTransformer.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/presets/no-babel/esmoduleTransformer'>;
}
declare module '@vue/cli-plugin-unit-jest/presets/no-babel/jest-preset.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/presets/no-babel/jest-preset'>;
}
declare module '@vue/cli-plugin-unit-jest/presets/typescript-and-babel/jest-preset.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/presets/typescript-and-babel/jest-preset'>;
}
declare module '@vue/cli-plugin-unit-jest/presets/typescript/jest-preset.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/presets/typescript/jest-preset'>;
}
declare module '@vue/cli-plugin-unit-jest/ui.js' {
  declare module.exports: $Exports<'@vue/cli-plugin-unit-jest/ui'>;
}
