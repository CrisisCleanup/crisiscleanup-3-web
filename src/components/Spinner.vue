<template>
  <div class="loader loader--style3 flex flex-col items-center" title="2">
    <svg
      id="loader-1"
      version="1.1"
      xmlns="http://www.w3.org/2000/svg"
      xmlns:xlink="http://www.w3.org/1999/xlink"
      x="0px"
      y="0px"
      :width="width"
      :height="height"
      viewBox="0 0 50 50"
      style="enable-background: new 0 0 50 50"
      xml:space="preserve"
    >
      <path
        :fill="color"
        d="M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615,14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z"
      >
        <animateTransform
          attributeType="xml"
          attributeName="transform"
          type="rotate"
          from="0 25 25"
          to="360 25 25"
          dur="0.6s"
          repeatCount="indefinite"
        />
      </path>
    </svg>
    <span v-if="message">{{ message }}</span>
  </div>
</template>

<script>
export default {
  name: 'Spinner',
  props: {
    width: {
      type: String,
      default: '40px',
    },
    height: {
      type: String,
      default: '40px',
    },
    color: {
      type: String,
      default: '#000',
    },
    message: {
      type: String,
      default: '',
    },
  },
};
</script>
