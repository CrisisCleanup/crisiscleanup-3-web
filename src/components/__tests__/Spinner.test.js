/**
 *
 * Tests for Spinner
 *
 * Components
 */

import { mount } from '@vue/test-utils';
import Spinner from '../Spinner.vue';

describe('Spinner', () => {
  it('should not log any errors', () => {
    const spy = jest.spyOn(global.console, 'error');
    mount(Spinner);
    expect(spy).not.toHaveBeenCalled();
  });

  it('should render correctly and match snapshot', () => {
    const wrapper = mount(Spinner);
    expect(wrapper.element).toMatchSnapshot();
  });
});
