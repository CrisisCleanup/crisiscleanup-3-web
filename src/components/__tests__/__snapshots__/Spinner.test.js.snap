// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render correctly and match snapshot 1`] = `
<div
  class="loader loader--style3 flex flex-col items-center"
  title="2"
>
  <svg
    height="40px"
    id="loader-1"
    style="enable-background: new 0 0 50 50;"
    version="1.1"
    viewBox="0 0 50 50"
    width="40px"
    x="0px"
    xml:space="preserve"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
    y="0px"
  >
    <path
      d="M43.935,25.145c0-10.318-8.364-18.683-18.683-18.683c-10.318,0-18.683,8.365-18.683,18.683h4.068c0-8.071,6.543-14.615,14.615-14.615c8.072,0,14.615,6.543,14.615,14.615H43.935z"
      fill="#000"
    >
      <animatetransform
        attributeName="transform"
        attributeType="xml"
        dur="0.6s"
        from="0 25 25"
        repeatCount="indefinite"
        to="360 25 25"
        type="rotate"
      />
    </path>
  </svg>
   
  <!---->
</div>
`;
